source/ceylon/test/engine/internal/file.ceylon

```diff
@@ -0,0 +1,39 @@
+import java.io {
+    File,
+    FileOutputStream,
+    PrintWriter
+}
+
+native class FileFacade(String path) satisfies Destroyable {
```

Comment:
oh, sorry, I didn’t even see your reply above… and no, your version is better, mine doesn’t create missing directories :)